exception CallException {
	1:i32 err_code,
	2:string message
}